applications/luci-app-opkg/htdocs/luci-static/resources/view/opkg.js

```diff
@@ -636,9 +636,9 @@ function handleInstall(ev)
 	    size;
 
 	if (pkg.installsize)
-		size = _('~%.1024mB installed').format(pkg.installsize);
+		size = _('~%1024mB installed').format(pkg.installsize);
```

Comment:
Should it not be `MiB`? Also would it not be better then to convert to Mb instead?